mongodbatlas/cloud_provider_snapshot_restore_jobs.go

```diff
@@ -202,3 +207,108 @@ func (s *CloudProviderSnapshotRestoreJobsServiceOp) Delete(ctx context.Context,
 
 	return resp, err
 }
+
+// ServerlessList gets all cloud provider snapshot serverless restore jobs for the specified cluster.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/cloud-backup/restore/return-all-restore-jobs-for-one-serverless-instance/
+func (s *CloudProviderSnapshotRestoreJobsServiceOp) ServerlessList(ctx context.Context, requestParameters *SnapshotReqPathParameters, listOptions *ListOptions) (*CloudProviderSnapshotRestoreJobs, *Response, error) {
+	if requestParameters.GroupID == "" {
+		return nil, nil, NewArgError("groupId", "must be set")
+	}
+	if requestParameters.ClusterName == "" {
```

Comment:
[nit] we should consider using `InstanceName`, for serverless they are not clusters

Reply:
changed :)